py3status/modules/scratchpad.py

```diff
@@ -94,7 +94,11 @@ def start(self):
         i3.main()
 
     def update(self, i3, event=None):
-        leaves = i3.get_tree().scratchpad().leaves()
+        scratchpad = i3.get_tree().scratchpad()
```

Comment:
can this return None?

Reply:
Yes. In current situation it doesn't matter. Code can handle it, but for better code maintainability I will add sanity check.